src/Microsoft.PowerPlatform.PowerApps.Persistence/Models/Text.cs

```diff
@@ -5,11 +5,11 @@
 
 namespace Microsoft.PowerPlatform.PowerApps.Persistence.Models;
 
-[FirstClass(nameof(Label))]
-public record Label : Control
```

Comment:
Why did you change from "label" to "text"? Just trying to follow the logic

Reply:
renamed in favor of the actual control we have on the portal